### PR TITLE
add sig-postgresql and initialized the doc

### PR DESCRIPTION
**Pull request by RogueJin**
add sig-postgresql and initialized the doc

### CODE REVIEW

**Comment by Mini256** [comment]
cc: @luzizhuo @tisonkun

File: special-interest-groups/sig-postgresql/membership.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "name": "sig-name",
```
Comment (Mini256) [inline special-interest-groups/sig-postgresql/membership.json:2]: Need to populate a sig name

File: special-interest-groups/sig-postgresql/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<!--
+The Special Interest Group Template is a template for starting new special interest groups in the GitHub organizations owned by PingCAP.  All special interest group, at minimum, must have the following files:
+
+* a `README.md` outlining the module that the SIG is responsible for, and the way this SIG is organized
+* a `roles-and-organization-management.md` outlining the responsibilities or institution specific to this SIG, as well as those that differ from [SIG Governance](sig-governance.md)
+* a `membership.md` with members of this SIG listed
+
+-->
```
Comment (tisonkun) [inline special-interest-groups/sig-postgresql/README.md:8]: Thanks for applying for new SIG @RogueJin ! You can remove all comments.

File: special-interest-groups/sig-postgresql/roles-and-organization-management.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Outline the organism specific to this SIG, as well as those that differ from [SIG Governance](sig-governance.md)
+-->
+
+## Join and Promotion
+
+Here we mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group), including common requirements, responsibilities and privileges of each community role.
+
+### Join
+
+You are welcomed to be a [contributor](../../architecture/README.md#contributor) once one of your PR about `postgresql` is merged.
```
Comment (tisonkun) [inline special-interest-groups/sig-postgresql/roles-and-organization-management.md:13]:
```suggestion
You are welcomed to be a [contributor](../../architecture/README.md#contributor) once one of your PR about TiDB for PG is merged.
```

It isn't about `postgresql`, is it?

File: special-interest-groups/sig-postgresql/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+- [DigitalChinaOpenSource](https://github.com/DigitalChinaOpenSource)
+  - [TiDB for PostgreSQL](https://github.com/DigitalChinaOpenSource/TiDB-for-PostgreSQL)    
+  - [DCParser](https://github.com/DigitalChinaOpenSource/DCParser)
+  - [dcbr](https://github.com/DigitalChinaOpenSource/dcbr)
+
+## Learning Materials
+
+<!-- Put the published documents of this Special Interest Group here -->
+-[TiDB for PostgreSQL—牛刀小试](https://zhuanlan.zhihu.com/p/379181280)
```
Comment (winkyao) [inline special-interest-groups/sig-postgresql/README.md:29]:
```suggestion
-[TiDB for PostgreSQL—牛刀小试(Chinese)](https://zhuanlan.zhihu.com/p/379181280)
```

File: special-interest-groups/sig-postgresql/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+-[TiDB for PostgreSQL 学习指南](https://zhuanlan.zhihu.com/p/383500491)
```
Comment (winkyao) [inline special-interest-groups/sig-postgresql/README.md:30]:
```suggestion
-[TiDB for PostgreSQL Guideline(Chinese)](https://zhuanlan.zhihu.com/p/383500491)
```

File: special-interest-groups/sig-postgresql/README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+- [DigitalChinaOpenSource](https://github.com/DigitalChinaOpenSource)
+  - [TiDB for PostgreSQL](https://github.com/DigitalChinaOpenSource/TiDB-for-PostgreSQL)    
+  - [DCParser](https://github.com/DigitalChinaOpenSource/DCParser)
+  - [dcbr](https://github.com/DigitalChinaOpenSource/dcbr)
+
+## Learning Materials
+
+<!-- Put the published documents of this Special Interest Group here -->
+-[TiDB for PostgreSQL—牛刀小试(Chinese)](https://zhuanlan.zhihu.com/p/379181280)
```
Comment (winkyao) [inline special-interest-groups/sig-postgresql/README.md:29]:
```suggestion
-[TiDB for PostgreSQL—Quick Start(Chinese)](https://zhuanlan.zhihu.com/p/379181280)
```

**Comment by tisonkun** [comment]
Looks good.

@winkyao would you ask for another TOC member to give a review?

**Review by winkyao — APPROVED** [review]
lgtm

**Review by zz-jason — APPROVED** [review]
LGTM

**Comment by sunxiaoguang** [comment]
LGTM

**Comment by RogueJin** [comment]
it still needs can-merge label, whom should be @ to add kind of label?

**Pull request merged**